sacred/utils.py

```diff
@@ -522,7 +556,7 @@ def format_filtered_stacktrace(filter_traceback="default"):
     elif filter_traceback in ("default", "always"):
         # print filtered stacktrace
         if sys.version_info >= (3, 5):
-            tb_exception = tb.TracebackException(
+            tb_exception = FilteredTracebackException(
                 exc_type, exc_value, exc_traceback, limit=None
             )
             return "".join(filtered_traceback_format(tb_exception))
```

Comment:
If this is a `FilteredTracebackException`, you can replace the call to `filtered_traceback_format` with `tb_exception.format()` and remove `filtered_traceback_format`. The code in `filtered_traceback_format` is a copy of `TracebackException.format` with filtering, but if the filtering is performed earlier, we can simply use `TracebackException.format()`. But, we should overwrite `format` in `FilteredTracebackException` to yield `"Traceback (most recent calls WITHOUT Sacred internals):\n"` instead of `'Traceback (most recent call last):\n'`.

Reply:
K.